crates/astria-sequencer/src/mempool.rs

```diff
@@ -117,6 +133,81 @@ impl std::hash::Hash for EnqueuedTransaction {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct TransactionExpired;
+
+impl std::fmt::Display for TransactionExpired {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Transaction expired")
+    }
+}
+
+const TX_TTL: Duration = Duration::from_secs(600); // 10 minutes 
+const REMOVAL_CACHE_SIZE: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(4096) }; // TODO make configuration variable 
```

Comment:
why not just use `usize`?

Reply:
Fraser recommended this instead I'm guessing for configuration safety purposes, but I can change back to just a `usize`